rules-tests/Php80/Rector/Class_/AnnotationToAttributeRector/FixtureAutoImported/skip_unique_entity.php.inc

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Rector\Tests\Php80\Rector\Class_\AnnotationToAttributeRector\FixtureAutoImported;
+
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\Table;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * @UniqueEntity(fields={"username"})
+ * @UniqueEntity("email")
+ */
+#[Entity(repositoryClass: 'UserRepository')]
+#[Table(name: 'users')]
+class User
+{
+}
+
+?>
```

Comment:
IMO to make this test work, there should be `@Entity` in the "before" part of fixture.
Now this file is ignored and skipped, so nothing can be imported here.

Bellow the `'Doctrine\ORM\Mapping\Entity'` is added, so it should be convered here too.

```php
AnnotationToAttributeRector::ANNOTATION_TO_ATTRIBUTE => ValueObjectInliner::inline([
                new AnnotationToAttribute('Doctrine\ORM\Mapping\Entity'),
```

Reply:
this fixture is for run the rector after Entity attribute exists, I will add one more fixture for non-exists Entity attribute

Reply:
I renamed fixture to `skip_unique_entity_when_entity_attribute_exists.php.inc`

Reply:
I added more fixture for do not remove `@UniqueEntity` when `#Entity` attribute not exists yet, with defined `@Entity` annotation so only change `@Entity` to `#Entity` annotation for it.

https://github.com/rectorphp/rector-src/pull/747/commits/69a49cff215c44134e86fe294a7ccadeae83e253